src/common/viz/FigureExtractor.js

```diff
@@ -19,7 +20,11 @@ define(['./Utils'], function (Utils) {
 
     FigureExtractor.prototype.extract = function(node) {
         const extractorFn = this.getMetaType(node);
-        return this[extractorFn](node);
+        if (!_.values(EXTRACTORS).includes(extractorFn)){
```

Comment:
I prefer `Object.values` since as [it is available in all currently supported web browsers](https://caniuse.com/#search=Object.values) and avoids an unnecessary dependency on a library.

Reply:
I have changed it to `Object.values` in 91b2652.